_posts/2021-08-25-please-manually-upgrade-ansible-and-remote-execution-plugins-on-25.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Please manually upgrade Ansible and Remote Execution plugins on 2.5
```

Comment:
Should we hint that it's Debian-only?

Reply:
And miss all the clicks? @melcorr would kill me :rofl:

Reply:
No, seriously, adding "Debian/Ubuntu" made the title even longer, and that felt wrong.

Reply:
Oh the other hand, it'd save a lot of RPM users a click.
```suggestion
title: [Debian] Please manually upgrade Ansible and Remote Execution plugins on 2.5
```

It may not be pretty, but it's not that much longer

Reply:
This does affect Ubuntu too ;)

@melcorr what do you think?

Reply:
If you drop the please, you have more real-estate.

Reply:
I thought about that too, but on the other hand it does indicate an action is required by the reader. Otherwise it could be a how to.

Reply:
Interesting perception. In IBM style, which is one of the authoritative texts on technical writing for decades, you never ever say "please". Using the imperative would have always been standard.

Reply:
Here's the quote, although i know we don't use it, it's just an FYI: 

please adv
Do not use in technical information. Terms of politeness are superﬂuous, convey the wrong tone for
technical material, and are not regarded the same way in all cultures. In marketing information, terms of
politeness might be appropriate.

Reply:
I get that, but this isn't purely documentation writing. Here we blast out some text and hope users do something with it. If you compare that to documentation, that's typically something people look up by themselves because they need information.

At a previous job we always had a styling for emails to indicate what it was. There was a clear header and a color. Red for action required, green for informational and blue for marketing (I may simplify and it has been a few years now). This helps to make it clear to the reader what you want from them.

I couldn't find a better word to indicate it here, but that's the reasoning I had.

Reply:
And I just see your quote now. I think this is closer to marketing information.

Reply:
> There was a clear header and a color. Red for action required, green for informational and blue for marketing (I may simplify and it has been a few years now).

We could add an actionrequired tag on the Discourse post when it is published, along with debian ubuntu labels. 
I wonder would that be a feature request for the RSS feed for Foreman, that we could somehow indicate urgency of the information. Most of the time it would not be needed, but it would have helped us here.

Reply:
Another way
```suggestion
title: [Debian] Ansible and Remote Execution plugins built on 2.5 with incorrect version
```

However, I'm not sure if that's better.

Reply:
I'd do something like :
```suggestion
title: [Action Required: Debian/Ubuntu] Upgrade Ansible and Remote Execution plugins on 2.5
```

Reply:
I think I'll go with Melanies last suggestion.